lib/plugins/aws/lib/naming.js

```diff
@@ -284,8 +284,15 @@ module.exports = {
   getMethodLogicalId(resourceId, methodName) {
     return `ApiGatewayMethod${resourceId}${this.normalizeMethodName(methodName)}`;
   },
-  getValidatorLogicalId(modelLogicalId) {
-    return `${modelLogicalId}Validator`;
+  getValidatorLogicalId() {
+    return `ApiGateway${_.upperFirst(
+      this.normalizeNameToAlphaNumericOnly(
+        _.camelCase(
```

Comment:
Is there a particular reason to use `_.camelCase` here? I don't think we use that anywhere else in `naming` module and I would probably just stick to `normalizeNameToAlphaNumericOnly` in that case

Reply:
I agree. I have removed it. Thanks for the suggestion.